src/mcts/params.cc

```diff
@@ -109,6 +109,15 @@ const OptionId SearchParams::kMultiPvId{
     "multipv", "MultiPV",
     "Number of game play lines (principal variations) to show in UCI info "
     "output."};
+const OptionId SearchParams::kHistoryFillId{
+    "history-fill", "HistoryFill",
+    "When to fill unknown history by repeating the earliest known position."};
```

Comment:
`Neural network uses 7 previous board positions in addition to the current one. During the first moves of the game such historical positions don't exist, but they can be synthesized. This parameter defines when to synthesize them (always, never, or only at non-standard fen position)`.